experimental/postcss-nesting/src/lib/merge-selectors/merge-selectors.ts

```diff
@@ -31,12 +31,6 @@ export default function mergeSelectors(node: Node, postcssResult: Result, fromSe
 					return false;
 				}
 
-				// see : https://github.com/w3c/csswg-drafts/issues/7979
-				if (parser.isPseudoElement(maybeCombinator)) {
-					startsWithCombinator = true;
-					return false;
-				}
```

Comment:
I am following what Chrome did and I agree with their analysis : https://github.com/w3c/csswg-drafts/issues/7979#issuecomment-1298337348